### ISSUE TITLE
VipsForeignLoad error on .svs file centOS 7 

### ISSUE THREAD

**Issue by RobRoss27** [libvips/libvips#1163]
I have installed vips from https://github.com/libvips/libvips/releases/download/v8.7.0/vips-8.7.0.tar.gz on a new centOS 7 server, so that `vips --version` results in `vips-8.7.0`.  I was unable to install three dependencies: gtk-doc, libwebpmux, and libimagequant, but I believe that I have the rest installed.  When I run: 
`vips dzsave filename.svs foldername.zip --background 0 --centre --layout google`, I get `VipsForeignLoad: "filename.svs" is not a known file format`.  

Any ideas on how I can fix this?
Thanks,
Rob

**Comment by lovell** [comment]
Is the `openslide-devel` package installed?

**Comment by RobRoss27** [comment]
I believe that it is installed.  Here's a jpg showing my screen when I try to install it.  
![openslide-devel-installed](https://user-images.githubusercontent.com/9118819/48413544-08ef5700-e716-11e8-8501-87852eb91127.jpg)

Also, here is a text file of the output when I run `./configure`.

[tempoutput.txt](https://github.com/libvips/libvips/files/2576229/tempoutput.txt)

Thanks!
Rob

**Comment by jcupitt** [comment]
Yes, looks like it found openslide. Perhaps you have two libvipses installed? I would check that the one you've built really is the one you are trying to run.

I'd test like this:

```
john@yingna ~/pics/openslide $ which vips
/home/john/vips/bin/vips
john@yingna ~/pics/openslide $ vipsheader CMU-1-Small-Region.svs 
CMU-1-Small-Region.svs: 2220x2967 uchar, 4 bands, rgb, openslideload
john@yingna ~/pics/openslide $ vips dzsave CMU-1-Small-Region.svs x
john@yingna ~/pics/openslide $
```

**Comment by jcupitt** [comment]
There's a dockerfile here that builds libvips + openslide on centos7.2:

https://github.com/jcupitt/docker-builds/blob/master/libvips-centos7.2/Dockerfile

I can run it like this:

```
john@yingna ~/pics $ docker run -it --rm -v /home/john/pics/openslide:/data libvips-centos7.2
[root@81e95aebe334 /]# cd /data/
[root@81e95aebe334 data]# vips dzsave CMU-1-Small-Region.svs x
[root@81e95aebe334 data]# 
```

And it seems to work.

**Comment by RobRoss27** [comment]
Thanks so much for your help.  For reasons I don’t understand, when I woke up this morning and ran the test that you suggested, it worked.  Further, when I tried to run the code which was giving me the error yesterday, I did not get the same error.  I don't recall changing anything which would have led to the change in behavior, but, for now, this issue seems resolved.  (If I get further errors that I can't solve, I'll let you know.)  I really appreciate your fast and effective help!

Rob
